Sources/EmbraceCore/Payload/Spans/SpanPayload.swift

```diff
@@ -37,7 +38,7 @@ struct SpanPayload: Encodable {
         self.spanId = span.spanId.hexString
         self.parentSpanId = span.parentSpanId?.hexString
         self.name = span.name
-        self.status = span.status.name
+        self.status = span.status == .ok ? span.status.name : (failed ? Status.sessionCrashedError().name : span.status.name)
```

Comment:
This will need to be broken out, 2 tertiary statements on the same line makes me cross-eyed.